### ISSUE TITLE
Error when trying to use helms3 on windows. [file does not exist]

### ISSUE THREAD

**Issue by mmarconm** [hypnoglow/helm-s3#157]
When i try to use the plugin for helm s3 on Windows 10 Pro, using Windows Terminal i got the following errors.

```sh
helm s3 init s3://my-bucket/charts
Error: exec: "C:\\Users\\mmarconm\\AppData\\Roaming\\helm\\plugins\\helm-s3.git/bin/helms3": file does not exist
```

I uninstalled the packages and helm several times and got the same error, even if a try to add a repo that are on s3 bucket with helm repo add, i got the same error.

**Comment by horacimacias** [comment]
I don't think this is currently being built for windows.
Perhaps you can try the following. 
1. Clone the git repo.
1. Build for windows:
```bash
GOOS=windows go build -ldflags "-X main.version=version" ./cmd/helms3 
```
1. put the resulting file (helms3.exe) on the path helm is complaining about so it's able to find the file.

Other than building for windows, which it builds for me, I've not tested the rest so you're likely to need some more fiddling (perhaps modifying plugin.yml to use windows paths, remove hooks and using `helm plugin install <path>` in order to install from our local path containing modified plugin.yml and produced binary)

**Comment by tensor-programming** [comment]
I was able to compile it on windows without much of an issue; seems like most of the modifications just require changes to the scripts.

**Comment by hypnoglow** [comment]
Hey folks,

Support for Windows is merged. I have no Windows device at the moment, so I would appreciate if someone could test it. I've tagged [this commit](https://github.com/hypnoglow/helm-s3/pull/168) `v0.12.0-test.1` which includes Windows support.

Could you try and see if it works for you?

```
helm plugin install https://github.com/hypnoglow/helm-s3.git --version 0.12.0-test.1
```

If everything is ok, I'll prepare release version soon.

**Comment by hypnoglow** [comment]
Support for Windows should be available in [v0.12.0](https://github.com/hypnoglow/helm-s3/releases/tag/v0.12.0) release.
